joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/IdentifierTests.scala

```diff
@@ -824,12 +824,6 @@ class IdentifierTests extends RubyCode2CpgFixture {
         .l
         .size shouldBe 1
     }
-    "recognise all call nodes" in {
```

Comment:
Was this test incorrect in the 1st place?

Reply:
Yes it was. I am not sure why it wasn't breaking